websocket-commons/src/main/java/org/lfenergy/compas/core/websocket/WebsocketSupport.java

```diff
@@ -45,4 +46,13 @@ public static <T> T decode(String message, Class<T> jaxbClass) {
         }
     }
 
+    public static void handleException(Session session, Throwable throwable) {
```

Comment:
Dit zou evt in 2 functies gesplitst kunnen worden (handleCompasException, handleException), dan bij de try/catch een catch voor CompasException. zo hoef je hier geen if-statement meer te gebruiken, maar heb je wel iets meer code-dupliacte

Reply:
Dan heb je wel het probleem dat je twee keer de ErrorResponse aan het maken bent. 
En bij the OnError krijg ik van de WebSocket gewoon een Throwable, dus ik denk dat in dit geval het op deze manier handiger is.